dryville_story.md

```diff
@@ -15,3 +15,14 @@ So let's say you made the big move. You and some friends have found your (desola
 From the founding day of Dryville, you need water. Drinking water is your first priority, as well as water to bathe in, clean dishes and clothes, and to wash your hands after you've cleaned that possum you caught for supper. And, of course, a toilet will come in very, very handy. So, your first priority will be to find a SOURCE of water. The obvious source is the creek or pond nearby. Maybe you can hire a beaver to create a dam in the creek and create a [lake](https://www.usgs.gov/special-topic/water-science-school/science/lakes-and-reservoirs) (actually, a reservoir).
 
 In your free time go ahead and start digging a hole in the ground for a [well](https://www.usgs.gov/special-topic/water-science-school/science/groundwater-wells). If you dig deep enough you might hit the [water table](https://www.usgs.gov/special-topic/water-science-school/science/aquifers-and-groundwater), where there will be standing water. You'll be able to lower a bucket down to get the available [groundwater](https://www.usgs.gov/special-topic/water-science-school/science/groundwater-what-groundwater). Things are starting out great -- you've already started using surface water (the creek and pond), ground water (your well), and you've even created your own water-storage system (the reservoir). So you now have a reliable source for your water needs. Time to relax? Not yet.
+
+## [Getting Water to Your Homes](https://www.usgs.gov/special-topics/water-science-school/science/water-density)
```

Comment:
I see what you've done here - added the section header and an appropriate link for "over 8 pounds a gallon", like #5 is requesting. It makes sense and the issue isn't very well worded. What we're actually looking for here is for your page to mimic the content [here](https://www.usgs.gov/special-topics/water-science-school/science/story-water-dryville) for the "Getting Water to Your Homes" section. This should include appropriate hyperlinks for "over 8 pounds a gallon" and any other links in the original text. Would you mind updating, please?